DBM-Naxx/ConstructQuarter/Patchwerk.lua

```diff
@@ -39,3 +46,36 @@ function mod:SPELL_MISSED(args)
 	end	
 end
 
+function mod:SPELL_AURA_APPLIED(args)
+	if args:IsSpellID(43093) then
+		warnGrievousThrow:Show(args.destName)
+		if self.Options.SetIconOnGrievousThrowTarget then
+			table.insert(GrievousThrowIcons, args.destName)
+			addIcons()
+		end
+	end
+end
+
+function mod:SPELL_AURA_REMOVED(args)
+	if args:IsSpellID(43093) and self.Options.SetIconOnGrievousThrowTarget then
+		removeIcons(args.destName)
+	end
+end
+
+
+local function addIcons()
+	for i,j in ipairs(GrievousThrowIcons) do
+		local icon = 9 - i
+		mod:SetIcon(j, icon)
+	end
+end
+
+local function removeIcons(target)
```

Comment:
Same as above, change all removeIcons into for example rIcons

Reply:
done